### DISCUSSION TITLE
Адаптация балуна под контент внутри

### DISCUSSION THREAD

**Discussion by unn3daby** [yandex-maps-unofficial/vue-yandex-maps#68] [Q&A (1.x)] [answered]
Привет! Подскажите пожалуйста, как адаптировать балун под внутренний контент?
![image](https://github.com/PNKBizz/vue-yandex-maps/assets/91330648/fa9ad7cf-ccbe-41fa-b608-1f0189499a43)
приходится ставить фиксированную высоту

**Comment by daniluk4000** [comment, marked as answer]
На данный момент приходится делать так, можно также ставить скроллбар.

В разработке версия 2.0 с Картами 3.0, там высота будет считаться автоматически так, как вы сами захотите - там эти попапы вы будете делать сами как компонент =)

**Reply by unn3daby** [reply]
Спасибо за ответ, поставил скроллбар
